springerlink.md

```diff
@@ -217,19 +211,13 @@ We make use of WAI-ARIA roles to support landmarks, and use WAI-ARIA roles and p
 <br><br>
 Some complex functionality may omit required WAI-ARIA roles, states, or attributes, or misuse WAI-ARIA markup and techniques. We are actively working to find and repair these problems. 
 <br><br>
-Multiple landmarks of the same type without unique labels are used on pages rendered in both designs. 
-<br><br>
-The SpringerLink checkout process pages are rendered in two separate designs. We are in the process of transferring all content from the older design to the newer design. 
-<br><br>
-Pages found at both checkout.springer.com and order.springer.com omit or use inappropriate headings and heading levels, including the use of unsemantic headings for visual effect. Multiple landmarks of the same type without unique labels are also found on these pages. 
-<br><br>
-Pages found at checkout.springer.com (<strong>older</strong> design) misapply HTML elements (e.g. label elements used for error messages), which may make it difficult for users of Assistive Technology to understand the structure and purpose of individual components. Some landmarks are incorrectly duplicated. 
+Multiple landmarks of the same type without unique labels are used on pages rendered in both designs.  
 <br><br>
-Pages found at order.springer.com (<strong>newer</strong> design) use form fields that rely solely on built-in browser error checking, which can lead to error messages being unavailable for users with older browser technology, or Assistive Technology that doesn't support these features. 
+Pages found at order.springer.com omit or use inappropriate headings and heading levels, including the use of unsemantic headings for visual effect. Multiple landmarks of the same type without unique labels are also found on these pages. 
 <br><br>
-Some custom interactive components found at checkout.springer.com (<strong>older</strong> design) are marked up unsemantically, which prevents them from being used by keyboard alone and from being recognised by Assistive Technology (e.g. the "login" and language selection components at the top of each page).
+Pages found at order.springer.com use form fields that rely solely on built-in browser error checking, which can lead to error messages being unavailable for users with older browser technology, or Assistive Technology that doesn't support these features. 
```

Comment:
Drop the "pages found at order.springer.com" part and combine with the paragraph above